images/miq-app/docker-assets/appliance-initialize.sh

```diff
@@ -11,8 +11,11 @@ sleep "${APPLICATION_INIT_DELAY}"
 # Prepare initialization environment
 prepare_init_env
 
+# Check Memcached readiness
+check_svc_status ${MEMCACHED_SERVICE_NAME} 11211
+
 # Check DB readiness
-check_db_status
+check_svc_status ${DATABASE_SERVICE_NAME} 5432
```

Comment:
The difference is that `pg_isready` will check both if the service is listening and also your connection parameters.

Seeing as we were not actually passing the full connection string they seem pretty equivalent to me.

I was curious so I went digging in the PG source. [This](https://github.com/postgres/postgres/blob/REL9_5_STABLE/src/interfaces/libpq/fe-connect.c#L2685) is the function that does the heavy lifting, the comments do a pretty good job of explaining what's going on.